slurmutils/models/option.py

```diff
@@ -48,6 +49,30 @@ def keys(cls) -> Iterable[str]:
             yield field.name
 
 
+@dataclass(frozen=True)
+class AcctGatherConfigOptionSet(_OptionSet):
+    """`acct_gather.conf` configuration options."""
+
+    EnergyIPMIFrequency: Callback = Callback()
+    EnergyIPMICalcAdjustment: Callback = Callback()
+    EnergyIPMIPowerSensors: Callback = Callback()
+    EnergyIPMIUsername: Callback = Callback()
+    EnergyIPMIPassword: Callback = Callback()
+    EnergyIPMIFrequency: Callback = Callback()
+    EnergyIPMITimeout: Callback = Callback()
+    ProfileHDF5Dir: Callback = Callback()
+    ProfileHDF5Default: Callback = Callback()
```

Comment:
[`ProfileHDF5Default`](https://slurm.schedmd.com/acct_gather.conf.html#OPT_ProfileHDF5Default) can also use `CommaSeparatorCallback`:

```suggestion
    ProfileHDF5Default: Callback = CommaSeparatorCallback
```